examples/simple-router/simple-router.rb

```diff
@@ -0,0 +1,167 @@
+require "pio"
+
+require "arp-table"
+require "interface"
+require "routing-table"
+require "router-utils"
+
+class SimpleRouter < Trema::Controller
+  include RouterUtils
+
+  def start( _args )
+    load "simple_router.conf"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.